python/lsst/dax/imgserv/version.py

```diff
@@ -1,4 +0,0 @@
-from pkg_resources import get_distribution
```

Comment:
Please don't delete this file.  I need this for things to work with the new packaging scheme.

Reply:
That file is not the one automatically generated by scons, that's the one I wrote.  If you run scons, it will generate one on top of it though (rewrite it) with different content.